Containerfile

```diff
@@ -0,0 +1,42 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest as BUILDER
+
+ENV OCS_CI_DIR=/opt/ocs-ci
+WORKDIR "${OCS_CI_DIR}"
+
+RUN dnf install --nodocs -y python38 python38-devel git libcurl-devel gcc openssl-devel libxml2-devel
+
+# Copy the entire source tree to the image
+# TODO: check to see if there are any more files we can drop from the image
+# TODO: the azure package is very big, we should consider to see how we can compact it
+COPY / "${OCS_CI_DIR}"
+
+RUN pushd "${OCS_CI_DIR}" \
+    && python3 -m venv venv \
+    && source venv/bin/activate \
+    && pip3 install --upgrade pip \
+    && pip3 install setuptools==65.5.0 \
```

Comment:
Why not to put setuptools in the requirements.txt?

Reply:
Not sure, this was based on the original work @OdedViner did [here](https://github.com/red-hat-storage/ocs-ci/tree/master/Docker_files/ocsci_container), so I just adapted a bit.

But I do think your point is valid, @OdedViner what was the reason?